feature-group-definitions/column-breaks.yml

```diff
@@ -0,0 +1,15 @@
+name: Column breaks
+description: The `break-after`, `break-before`, `break-inside` CSS properties control where columns start or end.
```

Comment:
Since `break-after` appears in the other description too, can you say something about the context this feature refers to? It's structurally similar to `gap` in grid and flex contexts, but I don't think we have good wording to copy from.

I think the context can be described as the use of `columns` or `column-count`.